src/main/groovy/com/moowork/gradle/node/npm/NpmInstallTask.groovy

```diff
@@ -59,7 +59,6 @@ class NpmInstallTask
 
     @OutputFile
     @Optional
-    @PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
Shouldn't this continue to be relative PathSensitive?

Reply:
I was reported as a warning in the [plugin build](https://travis-ci.org/node-gradle/gradle-node-plugin/builds/578305803#L253):

    Warning: Type 'com.moowork.gradle.node.npm.NpmInstallTask': property 'packageLockFileAsOutput' is annotated with @PathSensitive that is not allowed for @OutputFile properties.

Reply:
The [Gradle documentation](https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:up_to_date_checks) tells that it is only for inputs:
> Used with any input file property to tell Gradle to only consider the given part of the file paths as important. For example, if a property is annotated with `@PathSensitive(PathSensitivity.NAME_ONLY)`, then moving the files around without changing their contents will not make the task out-of-date.

Reply:
Oh right, I guess that's just for inputs